Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp

```diff
@@ -1412,10 +1442,103 @@ bool msblenContext::sendObjects(MeshSyncClient::ObjectScope scope, bool dirty_al
         eraseStaleObjects();
     }
 
+    //TODO-Sean Dillon: cleanup object instances array
```

Comment:
SceneExporter has the following callbacks, which we can use 

```
std::function<void()> on_prepare, on_success, on_error, on_complete;
```

Reply:
That should work perfectly, thank you!